scw/config.go

```diff
@@ -16,20 +18,64 @@ const (
 	defaultConfigPermission = 0600
 )
 
+const configFileTemplate = `# Scaleway configuration file
+
```

Comment:
missing `APIURL` and `Insecure`

Reply:
They are advanced features. IMO we can add [a link](https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#scaleway-config) to the config documentation but not add `APIURL` and `Insecure` by default.

Reply:
Not adding them here will break them

Reply:
`active_profile` is missing as well. The config should survive a read and a write operation. If this template does not include everything we can read from the config, then it will break on first `Save()`.

Reply:
Added

Reply:
@jerome-quere I think the point of @kindermoumoute is similar to the one I've made yesterday about dumping/loading again to guarantee that we have the same Config in and out of the function

Reply:
Yes we could update the test function to add a `yaml.Unmarshal` and a diff with config object

Reply:
Already done :)